### PR TITLE
Bug fixes0.1.1

### PR DESCRIPTION
**Pull request by NoaBablikiCertora**
=== BUG FIXES ===
1) send_only flag is automatically added to every conf file added to certora/conf directory
2) additional files that don't explicitly mention the main contract in the .conf file "file" section are now handled corretly

### CODE REVIEW

**Comment by NoaBablikiCertora** [comment]
* staging, rule_sanity can be added as additional flags

File: src/settings/RenamedMainWrapper.svelte [hunk 1]
```diff
@@ -67,7 +70,7 @@
         class="feedback-button"
         title="Feedback"
         href="https://forms.gle/zTadNeJZ7g1vmqFg6"
-        >Feedback
+        >Support
```
Comment (sarittenen) [inline src/settings/RenamedMainWrapper.svelte:73]: Feedback and Support

File: src/settings/validations/validators.js [hunk 2]
```diff
@@ -3,6 +3,32 @@
  *  consists of the relvant charecters.
```
Comment (sarittenen) [inline src/settings/validations/validators.js:3]: characters (typo)

File: README.md [hunk 3]
```diff
@@ -17,18 +17,18 @@ Content
...
 
-- Go to Definition
-- Call Trace & Variables View
+- Go to definition
+- Call trace & variables view
 - Check multiple contracts in parallel
 - Checking progress status
-- Support Multiple Jobs in parallel
-- New and Extended Settings Form
+- Support multiple jobs in parallel
```
Comment (sarittenen) [inline README.md:26]: multiple job runs in parallel

File: src/settings/validations/validators.js [hunk 4]
```diff
@@ -1,8 +1,34 @@
 /* ---------------------------------------------------------------------------------------------
  *  All validators functions. A validator is for checking if an input is valid,
- *  consists of the relvant charecters.
+ *  consists of the relevant characters.
  *-------------------------------------------------------------------------------------------- */
 
+import { solAdditionalContracts, solidityObj } from '../stores/store'
+
+function contractValidators() {
```
Comment (sarittenen) [inline src/settings/validations/validators.js:8]: has to be separated! should not define function inside return. with significant function names

**Pull request merged**